stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/reactionoptions/ReactionOptions.kt

```diff
@@ -96,11 +100,13 @@ internal fun DefaultReactionOptionItem(
     onReactionOptionSelected: (ReactionOptionItemState) -> Unit,
 ) {
     ReactionOptionItem(
-        modifier = Modifier.clickable(
-            interactionSource = remember { MutableInteractionSource() },
-            indication = rememberRipple(bounded = false),
-            onClick = { onReactionOptionSelected(option) }
-        ),
+        modifier = Modifier
+            .size(24.dp)
```

Comment:
Had to make this fixed size. Nothing should change its size when a popup is appearing. Otherwise it blinks especially if loading icons from the web.

Reply:
This is good - it makes sense and should be constrained to a 24-ish dp item.

Could we pull it out into a dimen item in the ChatDimens?